modules/docs/arrow-docs/docs/docs/arrow/data/listk/README.md

```diff
@@ -1,25 +1,22 @@
----
-layout: docs
-title: ListK
-permalink: /docs/arrow/data/listk/
-redirect_from:
-  - /docs/datatypes/listk/
----
-
 ## ListK
 
```

Comment:
If there are remaining tutorials in this page it should be added also in the listk header since the purpose of these changes is not maintaining file like these.

Reply:
Agree both need to be coherent

Reply:
I don't know if I've understood it, there isn't a tutorial in this Data type. Furthermore, it doesn't look well in the class header, I can show you if you want

Reply:
this is how it looks like in the kdoc y I add this block (I have changed the url to adapt to the new one):
![image](https://user-images.githubusercontent.com/20368422/65637244-2652aa00-dfe4-11e9-9300-00f66a032329.png)